PR 3 of n: Add inline comments where users need to replace code (#115)
AlexsLemonade/refinebio-examples: pathway-analysis/kegg_ortholog_mapping.Rmd

from the mouse gene symbols.

```{r}

> kegg_mouse_list[[grep("POLYMERASE", # Replace with desired pathway

Reviewer: ```suggestion
kegg_mouse_list[[grep("POLYMERASE", # Replace with an exact word or phrase that would filter in your desired pathway(s)
```

> kegg_mouse_list[[grep("POLYMERASE", # Replace with an exact word or phrase that would filter in your desired pathway(s)

Reviewer: ```suggestion
kegg_mouse_list[[grep("POLYMERASE", # Replace with a word (case sensitive) or phrase that would filter in your desired pathway(s)
```
